### PR TITLE
docs/TroubleShooting: add a solution of Kconfig-frontend issues

### PR DESCRIPTION
**Pull request by sunghan-chang**
This commit shows how to resolve a problem on mconf usage.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/TroubleShooting.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ## Common
 ### Trouble on using our toolchain
-When 64 bit machine tries to use 32 bit package like GDB, you can meet below:
```
Comment (girishks2000) [inline docs/TroubleShooting.md]: I think it should be GNU tool chain instead of our. We Samsung don't have any propreitory tool chain

Comment (sunghan-chang) [reply docs/TroubleShooting.md]: I missed ```our recommanded toolchain```. But anyway I modified. Thank you for comment.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**